Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -263,6 +277,9 @@ file that was distributed with this source code.
             })|raw }}
         </div>
     {% endif %}
+    {% if datepicker_use_button is not defined %}
+        {{ block('sonata_help') }}
```

Comment:
is this correct? why only show help if button is not defined?

Reply:
Indeed, care to explain?

Reply:
IMO you can remove the new if statement here, will it work @nykopol ?

Reply:
For the `DatePicker` and `DateTimePicker` types I explicitly disable the help message when there is a button (the calendar icon next to the form) because it break the rendering. 

For this two form type, this PR doesn't work as they corresponding template are in `sonata-project/core-bundle`. I'll do the corresponding PR if this one is accepted.